design/pipeline-artifacts.md

```diff
@@ -1,30 +1,30 @@
 # Pipeline Artifacts YAML shortcut
 
-**Status: Awaiting final review**
+**Status: Signed off**
 
-Pipeline Artifacts are the new way to move files between jobs and stages in your pipeline. They are hosted in Azure Artifacts and will eventually entirely replace FCS "Build Artifacts". Because moving files between jobs and stages is a crucial part of most CI/CD workflows, and because Pipeline Artifacts are expected to be the default way to do so, this spec proposes a YAML shortcut syntax for publishing and downloading artifacts.
+Pipeline Artifacts are the new way to move files between jobs and stages in your pipeline. They are hosted in Azure Artifacts and will eventually entirely replace FCS "Build Artifacts". Because moving files between jobs and stages is a crucial part of most CI/CD workflows, and because Pipeline Artifacts are expected to be the default way to do so, this spec proposes a YAML shortcut syntax for uploading and downloading artifacts.
 
-In this document, `artifact` refers specifically to publishing Pipeline Artifacts from the current pipeline. `downloadArtifact` refers to downloading Pipeline Artifact artifacts from the current pipeline, from other Azure Pipelines, and from other pipeline resources (e.g. Jenkins builds). 
+In this document, `artifact` refers specifically to uploading Pipeline Artifacts from the current pipeline. `download` refers to downloading Pipeline Artifact artifacts from the current pipeline and from other Azure Pipelines. Other pipeline systems (e.g. Jenkins) will be handled as [pipeline resources](pipeline-resources.md).
 
-Artifacts are distinct from other `resources` types, including `containers`, `repositories`, and `feeds`.
+Artifacts are distinct from other `resources` types, including `containers`, `repositories`, `packages`, and `feeds`.
 
-## Publishing artifacts: `artifact`
+## Uploading artifacts: `upload`
 
-`artifact` is a shortcut for the [Publish Pipeline Artifacts](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/publish-pipeline-artifact.md) task. It will publish files from the current job to be used in subsequent jobs or in other stages or pipelines.
+`upload` is a shortcut for the [Upload Pipeline Artifact](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/publish-pipeline-artifact.md) task (formerly called "Publish Pipeline Artifacts"). It will upload files from the current job to be used in subsequent jobs or in other stages or pipelines.
```

Comment:
what happens on retry? does it potentially dirty the same artifact? will it overwrite previously uploaded files?

Reply:
Yes, unless the user chooses to seal, they can update the artifact multiple times. This includes overwriting files uploaded in the first try or adding new files that weren't present in the first try.